src/public/js/app/progress_bars/traces/progress_bar_trace_view.js

```diff
@@ -0,0 +1,72 @@
+import { Controls } from "/js/app/resources/controls.js";
+import { ResourceSection } from "/js/app/resources/resource_section.js";
+import { AuthHelper } from "/js/auth_helper.js";
+import { apiUrl } from "/js/fetch_helper.js";
+import { ListenerOf, ReplicaListener } from "/js/lib/replica_listener.js";
+
+export class ProgressBarTraceView {
+    /**
+     * 
+     * @param {import("/js/app/progress_bars/traces/progress_bar_trace.js").ProgressBarTrace} progresBarTrace the progress bar trace to show
+     * @param {function() : any} onDelete the function to call after the progress bar trace is deleted
+     */
+    constructor(progresBarTrace, onDelete) {
```

Comment:
typo in name: `progresBarTrace` -> `progressBarTrace` (2 `s`)
will require multiple updates (i suggest F2 in IDE)